src/index.njk

```diff
@@ -26,6 +26,20 @@ masthead: true
         <p class="govuk-body govuk-!-margin-bottom-0"><a href="/patterns/" class="govuk-link govuk-!-font-weight-bold">Browse patterns</a></p>
       </div>
 
+      <div class="govuk-grid-column-full">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
```

Comment:
It appears that this is where that `govuk-!-margin-top-0` class that Chris found needs to go. Otherwise there's a bunch of extra space after the row with the 3 styles, components and patterns columns.

```suggestion
        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
```